### DISCUSSION TITLE
Feature request: Resizable GUI in Windows.

### DISCUSSION THREAD

**Discussion by chexmo** [xournalpp/xournalpp#2807] [Ideas]
Hello, in my honest opinion the GUI is too big for an average screen, hiding some buttons and occupying useful space.
I'd like to be resizable so everyone can find his/her preferred size for the GUI.
(With "GUI" I mean the buttons, toolbars and stuff, just in case this was needed to be said)

![imagen](https://user-images.githubusercontent.com/30608909/107894747-697e2780-6f0f-11eb-8373-6a019991867d.png)

**Comment by personalizedrefrigerator** [comment]
"View > Toolbars > Customize" might help!

**Reply by Technius** [reply]
That only changes what goes on the toolbars, not the actual size of the buttons. I think a better solution is to change some DPI scaling option in GTK, but I'm not sure where we would do that on Windows.

**Reply by chexmo** [reply]
I believe that a customizable factor for dpi would be a good approach. At least for now.
